src/modules/flow/converter/converter.c

```diff
@@ -822,8 +822,11 @@ drange_to_string_convert(struct sol_flow_node *node, void *data, uint16_t port,
     r = sol_flow_packet_get_drange(packet, &in_value);
     SOL_INT_CHECK(r, < 0, r);
 
-    r = do_float_markup(node, mdata->format, in_value, &out);
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wformat-nonliteral"
```

Comment:
no, this is not good as if the user provides a "%d %s" this thing will blow. I'd only be okay to allow this pragma if we add code to verify there is a single "%" and the type is of the expected type (ie: for integers it's 'd', 'x', 'i'...) Take care with things that would need more parameters, like "%.*d"  will need 2 parameters, one to define the precision, the other to do the actual number/value. And there is the %% to produce a % (escaping).